front50-core/src/main/groovy/com/netflix/spinnaker/front50/model/notification/Notification.groovy

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2016 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package com.netflix.spinnaker.front50.model.notification
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.netflix.spinnaker.front50.model.Timestamped
+
+class Notification extends HashMap<String, Object> implements Timestamped {
```

Comment:
Notification/Pipeline/PipelineStrategy controllers were previously dealing in `Map's`. 

This is a bit of a hack but I needed to have all objects being stored in S3 implement a common interface.